Cliqz/Extensions/AppDelegateExtension.swift

```diff
@@ -23,4 +23,13 @@ extension AppDelegate {
             }
         }
     }
+    
+    func customizeNnavigationBarAppearace() {
```

Comment:
I think the title of the function is a bit messed up :) "customizeNnavigationBarAppearace" should be "customizeNavigationBarAppearace". But it is ok.